src/components/AppNavigation/AppointmentConfigList/AppointmentConfigListItem.vue

```diff
@@ -73,6 +77,7 @@ export default {
 		ActionButton,
 		PencilIcon,
 		TrashCanIcon,
+	  CalendarCheckIcon,
```

Comment:
```suggestion
		CalendarCheckIcon,
```

Tabs and spaces are mixed.